L1Trigger/L1TTrackMatch/plugins/L1TrackerEtMissEmulatorProducer.cc

```diff
@@ -338,7 +338,7 @@ void L1TrackerEtMissEmulatorProducer::produce(edm::Event& iEvent, const edm::Eve
     edm::LogVerbatim("L1TrackerEtMissEmulatorProducer")
         << "====MET===\n"
         << "Integer MET: " << EtMiss.Et << "| Integer MET phi: " << EtMiss.Phi << "\n"
-        << "Float MET: " << (EtMiss.Et) * l1tmetemu::kStepMET
+        << "Float MET: " << (float)(EtMiss.Et) * l1tmetemu::kStepMET
```

Comment:
This should be changed from:
```c++
(float)(EtMiss.Et) * l1tmetemu::kStepMET
```
to
```c++
EtMiss.Et.to_double()
```

Reply:
Likewise, you need to change
```c++
(int)EtMiss.Et
```
on line 349 to
```c++
(int)EtMiss.Et.range()
```

Reply:
Fixed

Reply:
uhhh, did you push those fixes? I don't see them nor any commits added today.

Reply:
Pushed now